app/views/shops/index.html.erb

```diff
@@ -4,7 +4,7 @@
       <h1>All Shops</h1>
       <% @shops.each do |shop| %>
         <div id="shop">
-          <%= link_to shop.name, shops_path(shop) %>
+          <%= link_to shop.name, shop_path(shop) %>
```

Comment:
I won't comment on each instance, but any view that is using `shop_path(shop)` would end up using `shop_path(store: shop.slug)`